index.dd

```diff
@@ -98,8 +96,14 @@ void main()
 ----
 )
 
+) $(COMMENT your-code-here)
+))) $(COMMENT intro, div, div)
+
+$(COMMENT
+Picking the frontpage example after the page has loaded leads to reflows
+and re-rendering. Hence, we pick the example as soon as possible
 )
-)))
+<script src="$(ROOT_DIR)js/example_roulette.js"></script>
```

Comment:
> I still would like to add the example selection as well (so the file will grow)

I don't understand how adding the dropdown should affect this part. The dropdown can only work when JavaScript is enabled, so it would need to be added at the end, no?